README.md

```diff
@@ -140,27 +153,15 @@ Take surf_aircontrol_nbv for example: (You can use sm_mi to see this menu)
   
 </details>
 
+
 ## Credits
 
-Extensions used in this version:
-*   [SteamWorks](https://forums.alliedmods.net/showthread.php?t=229556) (KyleS)
-*   [SMJansson](https://forums.alliedmods.net/showthread.php?t=184604) (Thrawn2)
-*   [DHooks](https://forums.alliedmods.net/showthread.php?t=180114) (Dr!fter)
-*   [Discord API](https://github.com/Deathknife/sourcemod-discord) (Deathknife)
-*   [Trails Chroma](https://github.com/Nickelony/Trails-Chroma) (Nickelony)
-<details>
-  <summary>forked from fluffys - contributors</summary> 
-  
-*   Jonitaikaponi - Original ckSurf creator
-*   sneaK
-*   nikooo777 - ckSurf 1.19 Fork
-*   fluffys
-*   Jakeey802
+*   [Jonitaikaponi](https://github.com/jonitaikaponi) - Original ckSurf creator
+*   [sneaK](https://github.com/sneak-it)
+*   [nikooo777](https://github.com/nikooo777)
+*   [fluffys](https://github.com/fluffyst)
+*   [Jakeey802](https://github.com/Jakeey802)
 *   Grandpa Goose
-  
-</details>
-
-*	[Ace](https://github.com/13ace37) [xace.ch](https://xace.ch)
-*	[olokos](https://github.com/olokos) [Steam](https://steamcommunity.com/id/olokos/) [My server](https://kiepownica.pl/)
```

Comment:
You have removed both my steam link and my server link, leaving only github link, that anybody can get because I am already the owner of the repo, so the github link is really useless here, next to my name.

Reply:
done